forage-android/src/test/java/com/joinforage/forage/android/utils/UtilsTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.joinforage.forage.android.utils
+
+import com.joinforage.forage.android.generateJitter
+import kotlinx.coroutines.test.runTest
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+class UtilsTest {
+    @Test
+    fun `Jitter value should be in range -25 to 25`() = runTest {
+        val jitter = generateJitter()
+        assertThat(jitter).isGreaterThanOrEqualTo(-25)
+        assertThat(jitter).isLessThanOrEqualTo(25)
+    }
```

Comment:
this introduces randomness to our unit tests and may not catch issues with the `generateJitter`. Can we make this test deterministic?

```kotlin
internal fun generateJitter(random: Random = Random.Default): Int {
    return random.nextInt(-25, 26)
}

val fixedRandom = Random(12345) // Update seed value here
val jitter = generateJitter(fixedRandom)
```

Reply:
I'm going to seed the RNG at the method level so it will look something like:
```
internal fun getJitterAmount(): Int {
    val random = Random(999)
    return random.nextInt(-25, 26)
}
```

Reply:
[c5943c9](https://github.com/teamforage/forage-android-sdk/pull/94/commits/c5943c94e45dd5b967eff50e4cf70593a4c3eac7)

Reply:
fixing the seed at the method level will make the function return a fixed value each time.

As seen in this sandbox: https://pl.kotl.in/gX-Sm2V5A

Can we use `Random.Default` for the actual method and the fixed seed value for the test?